### DISCUSSION TITLE
Building mp for RPI-PICO-W

### DISCUSSION THREAD

**Discussion by beetlegigg** [micropython/micropython#14264] [RP2040 / Pico] [answered]
I've built mp a couple of times before and I'm following the same steps as previously and appear to be following the directions in the Readme of the RP2 port.   I'm doing this on a rpi 5 box where as I previously was using a rpi4.  
In the ports/RP2 dir I do 
$ make submodules
$ make clean
$ make
and this all appears to work OK.  I change dir to boards/RPI_PICO_W and in that dir I enter:
make BOARD=RPI_PICO_W submodules
But I just get the message:  make: *** No rule to make target 'submodules'.  Stop.
Any idea's what could be wrong and what I should do next?

**Comment by davefes** [comment]
Maybe `make BOARD=RPI_PICO_W`

**Comment by beetlegigg** [comment]
Thanks for the suggestion @davefes but leaving off the 'submodules'  give a lightly different message of:
make: *** No targets specified and no makefile found.  Stop

The Boards/RPI_PICO_W directory has the following files:
board.json  manifest.py  mpconfigboard.cmake  mpconfigboard.h  pins.csv

**Comment by beetlegigg** [comment, marked as answer]
I have figured it out, I should not be changing dir to boards/RPI_PICO_W but I should do all the make stuff from the ports/RP2 dir.  Ok its on with the show :-)
